src/config/2021.json

```diff
@@ -1287,6 +1287,17 @@
       ],
       "avatar_url": "60089915",
       "github": "ziemek-bucko"
+    },
+    "kartashovio": {
+      "name": "Timur Kartashov",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "60577846",
+      "website": "https://kartashov.io/",
+      "twitter": "krutoi_paren",
+      "github": "kartashovio",
+      "linkedin": "kartashov-io"
```

Comment:
We like these to be alphabetical order by first name to make them easier to find. Could you move it?

Reply:
Okay, done.